util/util.go

```diff
@@ -125,56 +122,6 @@ func Ordinalize(n int) string {
 	return fmt.Sprintf("%d%s", n, m[an%10])
 }
 
-// ModifyImageTag takes an imageName (e.g., repository/image:tag), and returns an image name with updated tag
-func ModifyImageTag(imageName, tagName string) (string, error) {
```

Comment:
If we move these functions out of the `util` package, that's an API breaking change for clients, I'd suggest to avoid this in v0.3.x, copy the functions and call the new ones in the new package from here. 

Add a comment on top of it that mentions that these are deprecated and have been moved.

Reply:
good point, didn't realize clients could be using that, will put them back with a deprecation comment